### PR TITLE
Add torchscript export for IntentSlotClassificationModel.

### PR DESCRIPTION
**Pull request by egorchakov**
A bit of a roundabout way of allowing `IntentSlotClassificationModel`s to be exported to torchscript. Not immediately sure how to do this in a cleaner way without adding/changing too much.

### CODE REVIEW

**Comment by egorchakov** [comment]
Is there a way to check what jenkins is failing with? It seems the details link points to a non-public IP.

File: tests/collections/nlp/test_nlp_exportables.py [hunk 1]
```diff
@@ -103,6 +104,28 @@ def test_IntentSlotClassificationModel(self, dummy_data):
             assert onnx_model.graph.output[0].name == 'intent_logits'
             assert onnx_model.graph.output[1].name == 'slot_logits'
 
+    @pytest.mark.run_only_on('GPU')
+    @pytest.mark.unit
+    def test_IntentSlotClassificationModel_export_to_torchscript(self, dummy_data):
```
Comment (drnikolaev) [inline tests/collections/nlp/test_nlp_exportables.py:109]: This method needs to be extended by some basic assertions like the one below, `test_TokenClassificationModel_export_to_onnx`

**Review by drnikolaev — CHANGES_REQUESTED** [review]
Please add assertions

File: tests/collections/nlp/test_nlp_exportables.py (hunk 1 above)
Comment (egorchakov) [reply tests/collections/nlp/test_nlp_exportables.py:109]: Added some of those. Though torchscript model internals seem to be a bit more difficult to inspect -- hopefully the assertions make sense.

**Review by drnikolaev — APPROVED** [review]

**Review by okuchaiev — APPROVED** [review]
lgtm, thanks!

**Pull request merged**